conteudo/introducao/iniciorapido.md

```diff
@@ -0,0 +1,337 @@
+---
+linkTitle: InicioRapido
+title: Inicio Rapido do Brigade
+description: Inicio Rapido do Brigade.
+section: introducao
```

Comment:
```suggestion
linkTítulo: InicioRapido
título: Início Rápido do Brigade
descrição: Início Rápido do Brigade.
sessão: Introdução
```

Reply:
fixed. I am keeping metadata titles in English.